src/popUp.js

```diff
@@ -1,28 +1,10 @@
 import React, { Component } from 'react';
 
 class PopUp extends Component {
-  constructor(props){
-    super(props)
-    this.close = this.close.bind(this)
-  }
-  state = {
-    display: "block",
-    title: "Henry's day at the park",
-    description: "A write up on Henry the Thirds trip to Central Park"
-  }
-
-  close() {
-
-    this.setState({
-      display: "none"
-    })
-    this.props.closePopUp()
-  }
-
   render() {
     return (
-      <div className="popUp" style={{display: this.state.display}}>
```

Comment:
There's no need to set the display here anymore since you conditionally render popup inside article already.